deploy/docker/cp-edge/sync-routes.py

```diff
@@ -245,24 +246,41 @@ def search_custom_domain_cert(domain):
         if not cert_path or not key_path:
                 cert_path = pki_default_cert
                 key_path = pki_default_cert_key
-        
+
         print('Certificate:Key for {} will be used: {}:{}'.format(domain, cert_path, key_path))
         return (cert_path, key_path)
 
-# FIXME: once we'll get more than one "system endpoint" - a list of such endpoints shall be moved to the configuration
-SYSTEM_ENDPOINTS={ "CP_CAP_SPARK": { 
-                        "value": "true", 
-                        "endpoint": str(os.environ.get("CP_CAP_SPARK_UI_PROXY_PORT", "8088")), 
-                        "endpoint_num":  str(os.environ.get("CP_CAP_SPARK_UI_PROXY_ENDPOINT_ID", "1000")),
-                        "friendly_name": "SparkUI" }}
+def read_system_endpoints():
+        system_endpoints = {}
+        with open(nginx_system_endpoints_config_path, 'r') as system_endpoints_file:
+                system_endpoints_list = json.load(system_endpoints_file)
+                for endpoint in system_endpoints_list:
+                        system_endpoints[endpoint['name']] = {
+                                "value": "true",
+                                "endpoint": str(os.environ.get(endpoint['endpoint_env'],
+                                                               endpoint['endpoint_default'])),
+                                "endpoint_num":  str(os.environ.get(endpoint['endpoint_num_env'],
+                                                                    endpoint['endpoint_num_default'])),
+                                "friendly_name": endpoint['friendly_name']
+                        }
+        return system_endpoints
+
+SYSTEM_ENDPOINTS = read_system_endpoints()
+
 def append_system_endpoints(tool_endpoints, run_details):
         if not tool_endpoints:
                 tool_endpoints = []
         system_endpoints_params = SYSTEM_ENDPOINTS.keys()
+        tool_endpoints_dict = {}
+        for endpoint in tool_endpoints:
+                endpoint_obj = json.loads(endpoint)
+                tool_endpoints_dict[endpoint_obj['name']] = endpoint_obj['nginx']['port']
```

Comment:
May the `endpoint_obj['nginx']` returns `None`? ( [check for that case](https://github.com/epam/cloud-pipeline/pull/1267/files#diff-cb34f51c1b0b798757a8b597c39e6f9fR354) )

Reply:
Done, checking if both `name` and `nginx` are presented in object after conversion from json